src/app/servicesModule/services/create-poll-storage.service.ts

```diff
@@ -142,6 +150,7 @@ export class CreatePollStorageService {
         // console.log('searchParam',searchParam)
         // searchParam.withResultSize(100);
         const searchResult = await this.searcher.search(searchParam.build());
+        console.log('searchResult', searchResult)
         if (searchResult.results.length > 0) {
           for (const resultItem of searchResult.results.reverse()) {
```

Comment:
This will create a new array to re-build a list with a different sequence. Do we really need to reverse()?